Greenshot/Drawing/CropContainer.cs

```diff
@@ -56,7 +56,15 @@ private void Init()
 		/// We need to override the DrawingBound, return a rectangle in the size of the image, to make sure this element is always draw
 		/// (we create a transparent brown over the complete picture)
 		/// </summary>
-		public override Rectangle DrawingBounds => new Rectangle(0,0,_parent?.Width??0, _parent?.Height ?? 0);
+		public override Rectangle DrawingBounds {
+			get {
+				if (_parent?.Image is Image image) {
+					return new Rectangle(0, 0, image.Width, image.Height);
+				} else {
+					return new Rectangle();

```

Comment:
I like Rectangle.Empty instead of contructing one.

Reply:
Ah ok, right